module/VuFind/config/module.config.php

```diff
@@ -674,8 +674,9 @@
     'Search/EditMemory', 'Search/Email',
     'Search/FacetList', 'Search/History', 'Search/Home', 'Search/NewItem',
     'Search/OpenSearch', 'Search/Reserves', 'Search/ReservesFacetList',
-    'Search/Results', 'Search/Suggest',
+    'Search/Results', 'Search/Suggest', 'Search/Versions',
     'Search2/Advanced', 'Search2/FacetList', 'Search2/Home', 'Search2/Results',
+    'Search2/Versions',
```

Comment:
Search2Controller extends AbstractSolrSearch, so as currently coded, it will not have access to the versionsAction(). The trait should either be added to Search2Controller or to AbstractSolrSearch if you want this to work.

Reply:
Looks like the Search2 Options class would also need getVersionsAction() to be defined.